storagenode/piecestore/endpoint.go

```diff
@@ -358,7 +359,13 @@ func (endpoint *Endpoint) Upload(stream pb.DRPCPiecestore_UploadStream) (err err
 			mon.IntVal("upload_failure_size_bytes").Observe(uploadSize)
 			mon.IntVal("upload_failure_duration_ns").Observe(uploadDuration)
 			mon.FloatVal("upload_failure_rate_bytes_per_sec").Observe(uploadRate)
-			endpoint.log.Error("upload failed", zap.Stringer("Piece ID", limit.PieceId), zap.Stringer("Satellite ID", limit.SatelliteId), zap.Stringer("Action", limit.Action), zap.Error(err), zap.Int64("Size", uploadSize), remoteAddrLogField)
+			if errors.Is(err, context.Canceled) {
+				// Context cancellation is common in normal operation, and shouldn't throw a full error.
+				endpoint.log.Info("upload failed", zap.Stringer("Piece ID", limit.PieceId), zap.Stringer("Satellite ID", limit.SatelliteId), zap.Stringer("Action", limit.Action), zap.Error(err), zap.Int64("Size", uploadSize), remoteAddrLogField)
```

Comment:
I would suggest going even farther and taking out the `zap.Error(err)` parameter. You could add `"(context canceled)"` to the log message itself instead. Our errors tend to be very long-winded and the stack trace is unlikely to be important.

Maybe we could log the error and stack trace in a Debug call below as well, just in case it ever _is_ important.

Same recommendation for the Download case.